schema/users.ts

```diff
@@ -75,6 +101,7 @@ export const examples: User[] = [
     createdAt: '2024-06-16T00:10:38.571Z',
     updatedAt: '2024-06-16T00:10:38.571Z',
     lastActiveAt: '2024-06-16T00:15:13.990Z',
+    badges: []
```

Comment:
nit: suggest provide a fixture with badges, so that we can verify if the zod schema and ES schema for badges are in sync.

It would be great if we can leave a fixture without `badges` to denote that it's valid for the field to be missing.